INSIGHTS-ONPREM.md

```diff
@@ -0,0 +1,34 @@
+# Insights OnPrem instructions for starting koku in development environment
+## Kafka
+We use the ros-ocp-backend kafka container. Start the ros-ocp-backend kafka service.
```

Comment:
can you add the command as an example? In case someone is not familiar on how to start the kafka service on their own

Reply:
fixed